README.md

```diff
@@ -36,127 +45,173 @@ Simply run `composer require mlocati/ip-lib`, or add these lines to your `compos
 
 ## Sample usage
 
+
+
 ### Parse an address
 
 To parse an IPv4 address:
 
 ```php
-$address = \IPLib\Address\IPv4::fromString('127.0.0.1');
+$address = IPv4::fromString('127.0.0.1');
 ```
 
 To parse an IPv6 address:
 
 ```php
-$address = \IPLib\Address\IPv6::fromString('::1');
+$address = IPv6::fromString('::1');
 ```
 
 To parse an address in any format (IPv4 or IPv6):
 
 ```php
-$address = \IPLib\Factory::addressFromString('::1');
-$address = \IPLib\Factory::addressFromString('127.0.0.1');
+use IPLib\Factory;
```

Comment:
I prefer to keep the fully qualified class names in examples, without `use` statements: newbies may not know where to place `use` statements, and if they place it directly inside functions their code will break. So they'll submit an issue in the IPLib project asking what they are doing wrong (it already happened).
That's why I prefer the fully-qualified class names in README.md.

Reply:
Hi @mlocati,

yes, that's certainly a matter of preference. The rationale was because, arguably, without FQDN the code looks more readable as you focus on the actual class name but I get your point.

Thanks,  
Adriano